wiki/en/Running-a-Server.md

```diff
@@ -125,185 +144,174 @@ To register with one of the Directories built into the Jamulus Client, replace `
-* **macOS users** - Double-click the "Jamulus Server" icon in Applications (assuming you put the files from the install there as per [these instructions](Installation-for-Macintosh)).
-* **Linux users** - Launch the "Jamulus Server" shortcut. Or you can open a terminal window (`CTRL+ALT+t` on Debian and related distros), type `jamulus -s` and hit return.
+## Other options
 
+{% include_relative Include-Shared-Commands.md %}
 
-## Server Setup
+---
 
-<figure><img src="{% include img/en-screenshots/server-window-setup.inc %}" style="width:80%; border:5px solid grey;" loading="lazy" alt="Image of the Jamulus Server setup window"></figure>
+## Recording
 
-### The Directory list
+##### `-R or --recording` 
+Set server recording directory; Server will record when a session is active by default. 
 
-**None**: By default, you will not be connected to a Directory and will be running in unregistered mode. [Read these instructions](#running-an-unregistered-server) to have other people connect to your Server in this mode.
+**Note:** You will need to save recordings to a path _outside_ of the jamulus home Directory, or remove `ProtectHome=true` from your systemd unit file (be aware that doing so is however a potential security risk).
 
-**Genre**: To allow other people to see your Server on one of the Directories built into the Client, select your desired genre Directory. You should see a confirmation message saying whether your Server has registered successfully. If not, and you leave your Server running, it will keep trying to register until a free slot becomes available.
+Recordings are per track in [Audacity](https://www.audacityteam.org/) `.lof` format and [REAPER](https://en.wikipedia.org/wiki/REAPER) `.rpp`. Open the respective files to listen to them in those applications.
 
-**Custom**: This allows you to specify a custom directory on which to be listed. See the "Options" tab for the Custom Directory address you want to use.
+Note that when your Server is recording, Clients will display a message that recording is on.
 
-To run your Server _as_ a Directory, you need to set the Custom Directory address as `localhost` or `127.0.0.1` and set the "Genre" to "Custom". [Read this guide](Directories) for further details.
+##### `--norecord` 
+Set server not to record by default when recording is configured.
 
-### My Server Info
 
-When running as a Registered Server this displays the Server's name, city and country so that other users can easily identify it in the Directory listing.
+### Controlling Recording
 
-### Chat Welcome Message
+Recording starts once the first person connects to the Server, and stops when the last person leaves. 
 
-The text entered here appears to all users when they join the Server (the chat window will open automatically for them). HTML is also supported.
+If the Server receives a SIGUSR1 signal during a recording, it will start a new recording in a new Directory. SIGUSR2 will toggle recording on/off.
```

Comment:
```suggestion
If the Server receives a SIGUSR1 signal during a recording, it will start a new recording in a new directory. SIGUSR2 will toggle recording on/off.
```

Our Capital-D Directory is something different.